crates/sui-graphql-rpc/src/extensions/logger.rs

```diff
@@ -140,28 +183,55 @@ impl Extension for LoggerExtension {
                             }
                         }
                     }
-                    error!(
-                        target: "async-graphql",
-                        "[Response] path={} message={}", path, err.message,
-                    );
+                    // in case of error, an error code is set which helps us
+                    // distinguish between INTERNAL, BAD REQUEST, BAD INPUT, etc.
+                    if let Some(ext) = &err.extensions {
+                        if let Some(code) = ext.get("code") {
+                            if let async_graphql_value::Value::String(val) =
+                                code.clone().into_value()
+                            {
+                                if val == code::INTERNAL_SERVER_ERROR {
```

Comment:
written in the previous PR but I'm guessing the code::INTERNAL_SERVER_ERROR is from errors module, in which case can we reference it with errors::code::INTERNAL_SERVER_ERROR to differentiate from ext.get("code")